.golangci.yml

```diff
@@ -358,7 +358,7 @@ linters:
         # All rules from candidate.yml
         - name: blank-imports
         - name: argument-limit
-          arguments: [5]
+          arguments: [8]
```

Comment:
If we have this many parameters, we should use different patterns like config structs or functional option parameters.